### PR TITLE
🌐 Add German translation for `docs/de/docs/tutorial/response-status-code.md`

### PR DESCRIPTION
**Pull request by nilslindemann**
← `tutorial/extra-models.md` (#10351)
→ `tutorial/request-forms.md` (#10361)

[German translation progress](https://github.com/tiangolo/fastapi/discussions/10582)

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 3dc76228ab0908e6601e0a72453f0ea90fd86605 at: https://4beca6c2.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit e3c7f1ed0a391627302ad7213aa0e7e1900b6117 at: https://4aa42ef4.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 825c3271cdfffa824b860e24052e219a61cd5a65 at: https://737db89a.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 9ff2ff86cc37a49eb3aef3fae71e9a8107da3c8d at: https://a7ae81c7.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 26255012fa15e1b68fe5c36ea5a9c0e9d8016deb at: https://ace6d452.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 1e1628333650e6d7390eccf63a477497ee4a7d15 at: https://4da35bac.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit c2b500b34c71bc97ed1f703e2a58b0a9bcf3a313 at: https://cfd70024.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 57824a59cd1d880455332b09404ff481dfae2493 at: https://af124631.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 7b4289629c5c115a909ddae1be2cbeedaaa6d71c at: https://992b1ac6.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit fb4897f6ce4ae34e358c7f9eb2429bdeed4ef7df at: https://8faba312.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit ebe343b722c49defab7a72b8dd52ae7350b52407 at: https://77dcaff1.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 5e20ffdd0b8e5b91e281264f9e5ce3e004f6d362 at: https://596db1aa.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 7dfa73fbc15889500768ee48bc0bec2068fd11f0 at: https://cfe373be.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit f10d2f94dd163f1350650639233ed46ddf5812c8 at: https://1285da69.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 7e6309265e87d0fad2c445bdc9938c929b07a0f5 at: https://a4dd19d0.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 74698ff16b205b49b470df0ed6c7494fb4ff21e2 at: https://350c3c05.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit d6667b2c2a33ba24f4b150cfaf993951348c21e2 at: https://b3c27a77.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 91c40adedd467372571d1770dfcd4e02ec7f882d at: https://69ea0c2c.fastapitiangolo.pages.dev

File: docs/de/docs/tutorial/response-status-code.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+* `@app.put()`
+* `@app.delete()`
+* usw.
+
+```Python hl_lines="6"
+{!../../../docs_src/response_status_code/tutorial001.py!}
+```
+
+!!! note "Hinweis"
+    Beachten Sie, dass `status_code` ein Parameter der „Dekorator“-Methode ist (`get`, `post`, usw.). Nicht der *Pfadoperation-Funktion*, so wie die anderen Parameter.
```
Comment (mattkoehne) [inline docs/de/docs/tutorial/response-status-code.md:16]: `so wie die anderen Parameter.` -> `so wie die anderen Parameter und der Body.`

File: docs/de/docs/tutorial/response-status-code.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    * `200` ist der Default-Statuscode, welcher bedeutet, alles ist „OK“.
+    * Ein anderes Beispiel ist `201`, „Created“ („Erzeugt“). Wird in der Regel verwendet, wenn ein neuer Datensatz in der Datenbank erzeugt wurde.
+    * Ein spezieller Fall ist `204`, „No Content“ („Kein Inhalt“). Diese Response wird verwendet, wenn es keinen Inhalt gibt, der zum Client zurückgeschickt wird, diese Response hat also keinen Body.
+* **`300`** und darüber steht für „Redirection“ („Umleitung“).  Responses mit diesen Statuscodes können einen oder keinen Body haben, mit Ausnahme von `304`, „Not Modified“ („Nicht verändert“), welche keinen haben darf.
+* **`400`** und darüber stehen für „Client error“-Responses („Client-Fehler“). Auch diese verwenden Sie am häufigsten.
+    * Ein Beispiel ist `404`, für eine „Not Found“-Response („Nicht gefunden“).
+    * Für allgemeine Fehler beim Client können Sie einfach `400` verwenden.
+* `500` und darüber stehen für Server-Fehler. Diese verwenden Sie fast nie direkt. Wenn etwas an irgendeiner Stelle in ihrem Anwendungscode oder im Server schiefläuft, wird automatisch einer dieser Fehler-Statuscodes zurückgegeben.
```
Comment (mattkoehne) [inline docs/de/docs/tutorial/response-status-code.md:55]: `in ihrem Anwendungscode` -> `in Ihrem Anwendungscode`

File: docs/de/docs/tutorial/response-status-code.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+Aber Sie müssen sich nicht daran erinnern, welcher dieser Codes was bedeutet.
+
+Sie können die Hilfsvariablen von `fastapi.status` verwenden.
+
+```Python hl_lines="1  6"
+{!../../../docs_src/response_status_code/tutorial002.py!}
+```
+
+Diese sind nur eine Annehmlichkeit und enthalten dieselbe Nummer, aber auf diese Weise können Sie die Autovervollständigung ihres Editors verwenden, um sie zu finden:
```
Comment (mattkoehne) [inline docs/de/docs/tutorial/response-status-code.md:78]: `ihres Editors` -> `Ihres Editors`

**Review by mattkoehne — CHANGES_REQUESTED** [review]
Thanks Nils :)

**Comment by nilslindemann** [comment]
Auch hier wieder ein herzliches Dankeschön fürs Review an dich, Matthias!

**Comment by tiangolo** [comment]
📝 Docs preview for commit 8a425cc405fcd38e9043c3a2adc28034dfec5c63 at: https://5e1781e6.fastapitiangolo.pages.dev

**Review by mattkoehne — APPROVED** [review]

**Pull request merged**